contrib/seeds/generate-seeds.py

```diff
@@ -85,32 +87,52 @@ def parse_spec(s, defaultport):
         (host,_,port) = s.partition(':')
 
     if not port:
-        port = defaultport
+        port = 0
     else:
         port = int(port)
 
-    host = name_to_ipv6(host)
+    host = name_to_bip155(host)
 
-    return (host,port)
+    return host + (port, )
 
-def process_nodes(g, f, structname, defaultport):
-    g.write('static SeedSpec6 %s[] = {\n' % structname)
-    first = True
+def ser_compact_size(l):
+    r = b""
+    if l < 253:
+        r = struct.pack("B", l)
+    elif l < 0x10000:
+        r = struct.pack("<BH", 253, l)
+    elif l < 0x100000000:
+        r = struct.pack("<BI", 254, l)
+    else:
+        r = struct.pack("<BQ", 255, l)
+    return r
+
+def bip155_serialize(spec):
+    '''
+    Serialize (networkID, addr, port) tuple to BIP155 binary format.
+    '''
+    r = b""
+    r += struct.pack('>B', spec[0].value)
```

Comment:
nit: the corresponding line in the test framework uses `B` instead of `>B`:

https://github.com/bitcoin/bitcoin/blob/80a699fda9ff1129546cabbf17e955680a1cc705/test/functional/test_framework/messages.py#L273

since this is duplicated code, I think it is good to keep it "the same" and since byte order is irrelevant for one-byte fields, maybe remove the `>` from here.

Reply:
Sure, will remove it. Myself I tend to use ">" symbolically in any kind of big-endian structure just in case I don't forget it when adding fields later.